test/unit/anchore_engine/util/test_rpm.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from anchore_engine.util.rpm import compare_versions
+
+
+class TestRpmVersionHandling(unittest.TestCase):
+    """
+    Tests for version comparisons of the rpm version parser and compare code.
+
+    """
+
+    def test_version_comparision(self):
+        test_epoch = [
+            ('1:0', '0:1', 1),
+            ('1:0', '1', -1),
+            ('1:2', '1', 1),
+            ('2:4.19.01-1.el7_5', '4.19.1-1.el7_5', 0),
```

Comment:
Please add comments here indicating that it is expected these are non-rpm-spec comparisons. An external viewer would say this is incorrect behavior, so as a note to ourselves indicate how/why these cases are correct.